src/Stats.js

```diff
@@ -1,13 +1,47 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import { retrieveLogDatabase, retrieveUserPR, updateUserPR } from './DatabaseFunctions';
-import { Header, Table, Button } from 'semantic-ui-react';
+import { Header, Table, Button, Icon } from 'semantic-ui-react';
+import { StandardTypeContext } from './Context.js';
+import { Chart } from './Chart';
 
 const defaultUserPRs = {
   Bench: {tested1RM: 0, estimated1RM: 0},
   Deadlift: {tested1RM: 0, estimated1RM: 0},
   Squat: {tested1RM: 0, estimated1RM: 0},
   Total: {tested1RM: 0, estimated1RM: 0}
-  }
+}
+
+const strengthStandards = {
```

Comment:
You can put these objects in the Constants.js file or even make a Constants folder with different files for different types of constants